README.md

```diff
@@ -6,8 +6,8 @@ This is the official operator to deploy and manage IBM block storage CSI driver.
 Supported container platforms:
   - OpenShift v4.2
```

Comment:
Are we going to support ocp 4.2? what about ocp 4.4?

Reply:
Updated